pytorch_lightning/profiler/base.py

```diff
@@ -96,7 +96,7 @@ def profile(self, action_name: str) -> None:
         finally:
             self.stop(action_name)
 
-    def profile_iterable(self, iterable, action_name: str) -> None:
+    def profile_iterable(self, iterable: Iterable, action_name: str) -> Generator:
```

Comment:
can we also add typing for the generator? i.e. what does it yield?

Reply:
Done !